src/lib/_imports/trumps/s-hero-banner.css

```diff
@@ -0,0 +1,65 @@
+/* Banner */
+/* Banner: Container */
+@media (--medium-and-above) {
+  .s-home-banner {
+    height: 55.5rem;
+    width: 100vw;
+    display: block;
+  }
+}
+
+@media (--medium-and-below) {
+  .s-home-banner {
+    height: auto;
+    display: grid;
+  }
+}
+
+.s-home-banner {
+  position: relative; /* for `position: absolute` children */
+}
+
+
+/* Banner: Overlay */
+@media (--medium-and-above) {
+  .s-home-banner .s-home-banner__overlay-message {
+    position: absolute;
+    height: auto;
+    width: 475px;
+    left: 116px;
+    bottom: 50px;
+
+  }
+}
+
+@media (--medium-and-below) {
+  .s-home-banner .s-home-banner__overlay-message {
+    position: relative;
+  }
+}
```

Comment:
I think you were right. I think I was thinking it would be relative > absolute for some reason? maybe because s-home-banner is always relative? I think I just missed it

Reply:
Fixed